vm/systemSmartContracts/auction.go

```diff
@@ -153,12 +159,27 @@ func (s *stakingAuctionSC) setConfig(args *vmcommon.ContractCallInput) vmcommon.
 		return vmcommon.UserError
 	}
 
-	// argument 5 is equal with epoch
-	s.eei.SetStorage(args.Arguments[5], configData)
+	s.eei.SetStorage(epoch, configData)
 
 	return vmcommon.Ok
 }
 
+func (s *stakingAuctionSC) checkConfigCorrectness(config AuctionConfig) error {
+	if config.MinStakeValue == nil {
+		return vm.ErrConfigIncorrect
```

Comment:
Strongly suggest to use 
```GO
return fmt.Errorf("%w for MinStakeValue", vm.ErrConfigIncorrect)
```

As to be able to easily debug in case of an error

Reply:
done